packages/bruno-app/src/components/CollectionSettings/Auth/OAuth2/AuthorizationCode/index.js

```diff
@@ -37,6 +37,7 @@ const OAuth2AuthorizationCode = ({ collection }) => {
           clientId,
           clientSecret,
           scope,
+          state,
```

Comment:
Should probably be also reset in `handlePKCEToggle` below.

Reply:
and declared above as well as in inputsConfig.js

Reply:
Fixed both reviews, thank for pointing out @pietrygamat

Reply:
@lohxt1 added in spread.